x-pack/plugins/apm/public/components/shared/Links/apm/service_overview_link.tsx

```diff
@@ -9,19 +9,34 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import React from 'react';
+import { useUrlParams } from '../../../../context/url_params_context/use_url_params';
+import { pickKeys } from '../../../../../common/utils/pick_keys';
+import { APMQueryParams } from '../url_helpers';
 import { APMLink, APMLinkExtendProps, useAPMHref } from './APMLink';
 
 interface ServiceOverviewLinkProps extends APMLinkExtendProps {
   serviceName: string;
 }
 
+const persistedFilters: Array<keyof APMQueryParams> = [
+  'latencyAggregationType',
+];
+
 export function useServiceOverviewHref(serviceName: string) {
-  return useAPMHref(`/services/${serviceName}/overview`);
+  return useAPMHref(`/services/${serviceName}/overview`, persistedFilters);
 }
 
 export function ServiceOverviewLink({
   serviceName,
   ...rest
 }: ServiceOverviewLinkProps) {
-  return <APMLink path={`/services/${serviceName}/overview`} {...rest} />;
+  const { urlParams } = useUrlParams();
+
+  return (
+    <APMLink
+      path={`/services/${serviceName}/overview`}
+      query={pickKeys(urlParams as APMQueryParams, ...persistedFilters)}
+      {...rest}
+    />
+  );
```

Comment:
Is `ServiceOverviewLink` used anywhere? If not, let's remove it. We decided to replace the components with hooks (like `useServiceOverviewHref`). I think it was an oversight that we forgot to delete this one when moving to the hook.

Btw. feel free to migrate other links from components to hooks :)

Reply:
Yes, it is used by `ServiceOverviewDependenciesTable`